scipy/sparse/_data.py

```diff
@@ -238,21 +247,25 @@ def _arg_min_or_max_axis(self, axis, op, compare):
             indices = mat.indices[p:q]
             am = op(data)
             m = data[am]
-            if compare(m, zero) or q - p == line_size:
-                ret[i] = indices[am]
+            if explicit:
+                if q - p > 0:
+                    ret[i] = indices[am]
```

Comment:
As indicated by @stefanv in https://github.com/scipy/scipy/pull/16467/#issuecomment-1563457909, we need to treat the case of an empty row or column (the missing `else` case here).

Reply:
One option is to return a sparse result.

Reply:
This looks like the appropriate solution to me. What do you think of it, @stefanv?

The COO format is probably appropriate here.

Reply:
Yes, I like that solution!